ui/apps/platform/src/utils/searchUtils.ts

```diff
@@ -314,7 +314,12 @@ export function addRegexPrefixToFilters(
 }
 
 // Uses the generatePath function from react-router in addition to adding the query params
-export const generatePathWithQuery = (pathTemplate, params, queryParams) => {
+// TODO: Fallback needed?
+export const generatePathWithQuery = (
+    pathTemplate: string,
+    params: Partial<Record<string, unknown>>,
```

Comment:
What do you think about `pathParams`
* parallel with `pathTemplate`
* contrast to `queryParams`

Reply:
Aha, great minds think alike! That is arg name in `generatePathWithScanConfig` function.

Reply:
I'll update and add the fallback, looks like this is only being used in one other place.